mpv.spec

```diff
@@ -95,15 +95,16 @@ waf configure \
     --disable-build-date \
     --enable-libmpv-shared \
     --enable-gpl3 \
-    --enable-sdl2
+    --enable-sdl2 \
+    --enable-encoding
```

Comment:
What you fink about --enable-encoding, (https://github.com/mpv-player/mpv/commit/d0238711dc776aeee2509452202ba4748f863ee4)
and add patch:
https://github.com/mpv-player/mpv/commit/160497b8ff316f5ddd4deb6198302e5abbe7a56b
https://github.com/mpv-player/mpv/commit/49431626cb1bde400ab6de6adc68ce39cdbbf6f8

like this: https://github.com/surfernsk/mpv.spec/commit/77fb0c2422bac7d546e9fca1cb0a585bb1d219d1